### PR TITLE
added turbinia logs to syslog

### PR DESCRIPTION
**Pull request by alimez**

### CODE REVIEW

File: turbinia/config/logger.py [hunk 1]
```diff
@@ -38,6 +38,10 @@ def setup(need_file_handler=True, need_stream_handler=True):
   # Eliminate double logging from root logger
   logger.propagate = False
 
+  # Add turbinia logs to syslog
+  syslog_handler = logging.handlers.SysLogHandler(address = '/dev/log')
```
Comment (hacktobeer) [inline turbinia/config/logger.py:42]: this will only work on linux, insert a runtime check to see which platform we run on and adjust address accordingly?

**Review by hacktobeer — APPROVED** [review]
small runtime nit, but lgtm.

**Comment by hacktobeer** [comment]
also, fix yapf and import error :)

File: turbinia/config/logger.py (hunk 1 above)
Comment (alimez) [reply turbinia/config/logger.py:42]: @rbdebeer Good point but is there any instance that we run Turbinia on non-linux environment?
Also, python's support for exporting logs to eventlogs isn't as good as Linux/Mac.

Comment (hacktobeer) [reply turbinia/config/logger.py]: Let's stick with this for now, in general most people will use the docker deployment. I'll put it on my list for Q1 to refactor as in general we should keep this as generic as possible, it's an external oss project.

**Comment by alimez** [comment]
@aarontp PTAL

File: turbinia/config/logger.py [hunk 2]
```diff
@@ -38,6 +39,12 @@ def setup(need_file_handler=True, need_stream_handler=True):
   # Eliminate double logging from root logger
   logger.propagate = False
 
+  # Add turbinia logs to syslog
+  syslog_handler = logging.handlers.SysLogHandler(address='/dev/log')
+  syslog_handler.formatter = logging.Formatter(
+      'Turbinia: { "loggerName":"%(name)s", "levelNo":"%(levelno)s", "levelName":"%(levelname)s", "message":"%(message)s"}'
```
Comment (aarontp) [inline turbinia/config/logger.py:45]: Minor nit:  It looks like the line length is too long here.

Comment (alimez) [reply turbinia/config/logger.py:45]: Yapf formatted it this way. lmk if you want me to change that

Comment (aarontp) [reply turbinia/config/logger.py:45]: Yeah, yapf doesn't know how to break apart strings, but you can just do it manually like:
```
logging.Formatter(
    'foo '
    'bar')
```

If you don't break it up you could also add a lint disable comment, but breaking it up is probably better.

Comment (alimez) [reply turbinia/config/logger.py]: Done

**Review by aarontp — APPROVED** [review]
LGTM

**Comment by aarontp** [comment]
Looks like YAPF tests are still failing?   If you've already run yapf and still seeing this, make sure that you have the latest version installed locally.  I think the other failures are due to module dependency issues (I have a separate PR for this).

**Comment by alimez** [comment]
@aarontp Please merge

**Pull request merged**